docs/resources/workspace.md

```diff
@@ -14,11 +14,12 @@ resource "powerbi_workspace" "myworkspace" {
 #### The following arguments are supported:
 <!-- docgen:NonComputedParameters -->
 * `name` - (Required, Forces new resource) Name of the workspace.
+* `capacity_id` - (Optional) Capacity ID to be assigned to workspace.
```

Comment:
Would be good to indicate what prerequisites and permissions are required to be able to use capacity_id

Reply:
Updated the workspace.md with the minimum required permissions.